docs/pages/clients/registry.md

```diff
@@ -8,17 +8,16 @@ toc: false
 
 # Singularity Registry Client
 
-A Singularity Registry backend is optimized to interact with a Singularit Registry, the configuration which is set in a credentials file typically located at `$SREGISTRY_CLIENT_SECRETS` that defaults to `$HOME/.sregistry`. If you aren't familiar with a Singularity Registry, it is an open source effort to empower institutions to host their own registries. It's more substantial then this `sregistry` client as it involves a bit of set up and configuration, and must run on a server with Docker. To read more, [view the documentation here](https://singularityhub.github.io/sregistry). You should check out the [getting started](../getting-started) pages for a list of environment variables that you can configure, and the [global commands](../getting-started/commands.md) for commands that are useful for all clients. Here we will review the set of commands that are specific to the Singularity Registry client:
+A Singularity Registry backend is optimized to interact with a Singularity Registry, whose configuration is determined by the `SREGISTRY_REGISTRY_BASE`, `SREGISTRY_REGISTRY_USERNAME` and `SREGISTRY_REGISTRY_USERTOKEN` environment variables. If you aren't familiar with a Singularity Registry, it is an open source effort to empower institutions to host their own registries. It's more substantial then this `sregistry` client as it involves a bit of set up and configuration, and must run on a server with Docker. To read more, [view the documentation here](https://singularityhub.github.io/sregistry). You should check out the [getting started](../getting-started) pages for a list of environment variables that you can configure, and the [global commands](../getting-started/commands.md) for commands that are useful for all clients. Here we will review the set of commands that are specific to the Singularity Registry client:
 
  - *pull*: `[remote->local]` is a common use case. It says "get this remote registry image and pull it from there to my storage."
  - *push*: `[local->remote]` takes an image on your host and pushes to the registry (if you have permission).
  - *search*: `[remote]`: list containers for a remote endpoint, optionally with a search term.
  - *delete*: `[remote]`: delete an image from a remote endpoint if you have the correct credential.
 
-If you are using `sregistry` for a Singularity Registry, Python 3.3+ is required.
```

Comment:
Why did you take this out? I think it's still the case, although I shudder to think of someone using Python 2, given the whole end of life thing :)

Reply:
Because we stopped supporting python 2.7 in release 0.2.0 😉

Reply:
Yeah I probably should add some (global) statement that Python 3.x is required. This is good to remove then.

Reply:
Actually you may want to add some instructions on the main README.md on how to install it with pip and Conda...

Reply:
Would you care to add here?

Reply:
Sure, I can do that... commit coming up